cli/azd/internal/scaffold/spec.go

```diff
@@ -98,27 +98,56 @@ type ServiceSpec struct {
 	Name string
 	Port int
 
-	Env map[string]string
+	EnvironmentVariableInformation EnvironmentVariableInformation // todo: merge EnvironmentVariableInformation and env
 
 	// Front-end properties.
 	Frontend *Frontend
 
 	// Back-end properties
 	Backend *Backend
+}
 
-	// Connection to a database
-	DbPostgres    *DatabasePostgres
-	DbMySql       *DatabaseMySql
-	DbRedis       *DatabaseRedis
-	DbCosmosMongo *DatabaseCosmosMongo
-	DbCosmos      *DatabaseCosmosAccount
-
-	// AI model connections
-	AIModels []AIModelReference
+type EnvironmentVariableInformation struct {
+	StringEnvironmentVariables    []StringEnvironmentVariable
```

Comment:
We can add 2 types here: 
1. `StringEnvironmentVariables`, tits value will be quoted in resources.bicept. 
2. Another type: `VariableEnvironmentVariables`, it's value will not be quoted. 

To keep it simple, I didn't add the second type. To use a variable, I use `${variableName}` instead.